### PR TITLE
LPS-76325 SF - renaming.

### PR DESCRIPTION
**Pull request by brandizzi**

### CODE REVIEW

**Comment by brandizzi** [comment]
ci:test <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: modules/apps/forms-and-workflow/calendar/calendar-test/src/testIntegration/java/com/liferay/calendar/upgrade/v1_0_4/test/UpgradeClassNamesTest.java [hunk 1]
```diff
@@ -222,6 +222,15 @@ protected void addAssetVocabulary(long classNameId) throws Exception {
...
+	protected ClassName addClassName(String value) {
+		ClassName className = ClassNameLocalServiceUtil.createClassName(
+			CounterLocalServiceUtil.increment());
+
+		className.setValue(value);
+
+		return ClassNameLocalServiceUtil.addClassName(className);
+	}
+
 	protected ResourcePermission addResourcePermission(
```
Comment (brandizzi) [inline modules/apps/forms-and-workflow/calendar/calendar-test/src/testIntegration/java/com/liferay/calendar/upgrade/v1_0_4/test/UpgradeClassNamesTest.java]: Hey @SamZiemer, great work! Thanks!

I was getting problems  running tests twice. Apparently `ClassNameLocalService` has some kind of internal cache that was breaking tests running after the first one. It was quite weird, I guess it is very specific for class names. Anyway, I could circumvent this problem here.

Besides that, I did some small source formatting and little changes in other commits. As I've said before, these upgrade tests are somewhat complicated but always prove themselves useful in the worst time, so many thanks for your work again!

**Comment by brandizzi** [comment]
ci:reopen

**Comment by inacionery** [comment]
ci:test

**Comment by brianchandotcom** [comment]
Merged. Thank you.
View total diff: https://github.com/brianchandotcom/liferay-portal/compare/123f121c9a...c2fbd4eacf

**Pull request closed without merging**